Sources/SwiftMemcache/Extensions/ByteBuffer+SwiftMemcache.swift

```diff
@@ -101,6 +101,22 @@ extension ByteBuffer {
                 self.writeInteger(UInt8.R)
             }
         }
+
+        if let arithmeticMode = flags.arithmeticMode {
```

Comment:
I guess you can't specify a storage mode and arithmetic mode. Can we add a precondition somewhere to check for this